packages/components/src/Truncate/Truncate.test.js

```diff
@@ -61,6 +61,20 @@ describe('Truncate component', () => {
                     const wrapper = shallow(<Truncate text={text} inline={isInline} length={1000} />);
                     expect(toJson(wrapper)).toMatchSnapshot();
                 });
+
+                it('hovering over toggles to collapse', () => {
```

Comment:
This test could be split up in two tests to separate the testing of mouseenter and mouseleave.